utils/db_test.ts

```diff
@@ -71,7 +72,14 @@ Deno.test("[db] (get/create)Item()", async () => {
 
   assertEquals(await getItem(item.id), null);
 
+  const itemsCount =
+    (await getAnalyticsMetricPerDay("items_count", new Date()))?.valueOf() ??
+      0n;
   await createItem(item);
+  assertEquals(
+    (await getAnalyticsMetricPerDay("items_count", new Date()))!.valueOf(),
```

Comment:
Is `.valueOf()` needed? Ditto elsewhere.

Reply:
It seems so. This is the test result when I remove the velueOf():

[Diff] Actual / Expected
-   [Deno.KvU64: 3n]
+   3n

Reply:
I see. Fair enough.